### PR TITLE
📪📫📬📭💌 modify home address 💌📭📬📫📪

### PR DESCRIPTION
**Pull request by roc**
- Changes to expect 4 address lines, a postcode and a country

### CODE REVIEW

File: src/main/resources/evw/evw-schema.json [hunk 1]
```diff
@@ -197,7 +197,11 @@
                             "type": "string"
                         },
                         {
-                            "id": "4",
+                            "id": "postcode",
```
Comment (ryandac) [inline src/main/resources/evw/evw-schema.json]: I didn't think you can have named id's in an array. Can you?

Comment (roc) [inline src/main/resources/evw/evw-schema.json]: @ryandac I think you can have whatever you like as long as it doesn't match another id within the array of objects. Generally speaking I guess `id` is normally an incremented `int` value, but we're not doing that either. Shall I change it to be numbers wrapped in strings to match our conventions? I'm totally easy about this btw

Comment (roc) [inline src/main/resources/evw/evw-schema.json]: I'm using http://json-schema.org/example1.html as my reference here, I'm definitely not a reliable JSON schemas person lol
> Set of products:
```
[
    {
        "id": 2,
        "name": "An ice sculpture",
        "price": 12.50,
        "tags": ["cold", "ice"],
        "dimensions": {
            "length": 7.0,
            "width": 12.0,
            "height": 9.5
        },
        "warehouseLocation": {
            "latitude": -78.75,
            "longitude": 20.4
        }
    },
    {
        "id": 3,
        "name": "A blue mouse",
        "price": 25.50,
        "dimensions": {
            "length": 3.1,
            "width": 1.0,
            "height": 1.0
        },
        "warehouseLocation": {
            "latitude": 54.4,
            "longitude": -32.7
        }
    }
]

```

**Comment by ryandac** [comment]
schema and example validated against http://www.jsonschemavalidator.net/

**Comment by ryandac** [comment]
Maybe we should have swapped the positioning of the postcode and country around.

**Comment by roc** [comment]
@ryandac Smashed it

**Pull request merged**